app/src/main/res/layout/fragment_get_started.xml

```diff
@@ -70,6 +70,7 @@
             android:id="@+id/group_tool_bar"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
+            android:visibility="@{getStartedVM.isEnableBackButton()}"
```

Comment:
```suggestion
            android:visibility="@{viewModel.isEnableBackButton()}"
```

Reply:
Cái này nó trùng với tên viewModel e để ở Base thì mỗi lần lại phải gọi this@..Fragment.viewModel à a